src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -219,7 +219,7 @@ def convert_size(value, units='b'):
         return str(val) + new
     except KeyboardInterrupt:
         raise
-    except:
+    except Exception:
```

Comment:
We still need to handle KeyboardInterrupt independent of other exceptions so that when the timer pops we don't ignore it and keep running.

Reply:
I don't think you do any more.  KeyboardInterrupt inherits BaseException, not Exception.  You are now doing an except Exception instead of a blanket except.  A KeyboardInterrupt will be unhandled.

Reply:
You lost me. In the event of a KeyboardInterrupt it will throw an error that will eventually be caught somewhere up the stack. In the event of any other exception, the function returns None.

Reply:
`except Exception` does not catch every exception.
As seen here: https://docs.python.org/2/library/exceptions.html#exception-hierarchy
```
BaseException
 +-- SystemExit
 +-- KeyboardInterrupt
 +-- GeneratorExit
 +-- Exception
```
Everything in `Exception` will be caught, and that excludes KeyboardInterrupts.

Reply:
Point take. So is the request to remove the except for KeyboardInterrupt now that we're no longer using blanket excepts?

Reply:
Exactly.